tests/drivers/build_all/modem/uart.dtsi

```diff
@@ -45,3 +45,8 @@ test_quectel_bg9x: quectel_bg9x {
       mdm-power-gpios = <&test_gpio 0 0>;
       mdm-reset-gpios = <&test_gpio 0 0>;
 };
+
+test_gsm_ppp: gsm_ppp {
```

Comment:
`s/gsm_ppp/gsm-ppp/`

Reply:
Don't really understand, I pretty much followed the conventions of other modems defined above.

Reply:
I didn't notice that. So just leave it as is.